### PR TITLE
Public navigation

### PR DESCRIPTION
**Pull request by philmillr**
Created public navigation (Overview page, How to page, and University selection page) and implemented navigation bar. Content for pages to come (currently blank). Copied the CSS from Product 1.

The navigation bar allows one to navigate to the home page (which is pictured below), the how-to page (which is currently blank), and the University selection page (by clicking "get started")(this page currently just includes a button to navigate to the product 2 login page, or an unlinked button that will navigate to our product 1 login page). A screen shot of the university selection page is also attached below

Screen Shot 2024-03-13 at 7 20 25 PM

Screen Shot 2024-03-13 at 7 23 01 PM

### CODE REVIEW

**Comment by Emerson-Wenzel** [comment]
Great job! I appreciate the initiative you've taken with this and the fact that you've added the CSS already!

- please run the linter: https://docs.google.com/document/d/1tOsL0x2yXYmezNgGicf92E7V5TRr0Ew8u4bBbuAeCO4/edit#heading=h.8kt70r1a43m3 to help clean up the code
- please attach screenshots or a video to the description to make it easier for reviewers to understand the code changes made.
- If this is the navigation bar that will be used across the entire website, please assign yourself the task in Trello and mark it as WIP. That way no one accidentally duplicates your work :) CC @HongjieWang98 
- Just to confirm, so we will make this the new "home page", and then from here we will redirect to the legacy public page or the new login/signup  page, is that correct? CC @kevinbae15  for domain name related changes.
- you will need to rebase to fix merge issues on App.jsx . Please let me know via Messenger if I can help with that.

**Review by HongjieWang98 — COMMENTED** [review]
Thank you, Phil, for implementing the public navigation and adding the CSS. +1 to Emerson's comment.

Regarding the code rebase, here are some steps you can follow for reference:

1. Fetch the changes in remote branch to local branch
2. Merge the changes in remote branch to local branch
3. Push the local branch again

**Comment by philmillr** [comment]
Thank you for your feedback @Emerson-Wenzel  and @HongjieWang98 ! My responses to your comments are below: 

1. Linter has been run! - currently debugging one prop issue and then it should be good to go!
2. Description of the PR has been added with not 1 but 2 screenshots!
3. There will be 2 navigation bars - one for once you've logged in and one for the public page. This is the external public navigation bar. I have created a ticket for each. The external public bar is in WIP and assigned to me. 
4. I will coordinate with @kevinbae15 about linking to the legacy public pages. However, I want to redo a significant chunk of the legacy pages. Do we want to combine these two code bases and then I can just edit it here? Or do you just want to link it, and I edit the old code base? 
5. Resolved the merge issue with App.jsx! (it was super minor lol)

Once I resolve this last linter issue, I will resubmit the PR!

**Review by HongjieWang98 — APPROVED** [review]

**Comment by HongjieWang98** [comment]
LGTM
There is a branch conflict need to be resolved before merge.

File: src/App.jsx [hunk 1]
```diff
@@ -2,15 +2,35 @@ import './App.css';
...
+        </Route>
+        <Route path="/Home/howto" element={<HowToPage />}>
+          {' '}
+        </Route>
+        <Route path="/Home/schoolnav" element={<SchoolNavPage />}>
+          {' '}
+        </Route>
+      </Routes>
+
+      <SellProvider>
```
Comment (kevinbae15) [inline src/App.jsx]: delete this
```suggestion
```

**Review by kevinbae15 — APPROVED** [review]

**Pull request merged**